src/main/java/appeng/parts/reporting/ConversionMonitorPart.java

```diff
@@ -69,27 +100,28 @@ public boolean onPartActivate(Player player, InteractionHand hand, Vec3 pos) {
             return true;
         }
 
-        if (!this.getMainNode().isActive()) {
```

Comment:
Please undo all the formatting changes, they make the diff quite harder to review.